test/attr/testable.swift

```diff
@@ -1,14 +1,12 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-swift-frontend -emit-module -enable-testing -o %t %S/../Inputs/empty.swift
-// RUN: %target-swift-frontend -typecheck -I %t -I %S/Inputs/custom-modules %s -verify
-// RUN: %target-swift-frontend -typecheck -I %t -I %S/Inputs/custom-modules %s -disable-testable-attr-requires-testable-module -DIMPORTS_ONLY
+// RUN: not %target-swift-frontend -typecheck -I %t -I %S/Inputs/custom-modules %s -DIMPORTS_ONLY 2>&1 | %FileCheck %s
+// RUN: %target-swift-frontend -typecheck -I %t -I %S/Inputs/custom-modules %s -disable-testable-attr-requires-testable-module -DIMPORTS_ONLY 2>&1 | %FileCheck %s --check-prefix DISABLED --allow-empty
 
-@testable import Swift // expected-error {{module 'Swift' was not compiled for testing}}
+// CHECK: module 'Swift' was not compiled for testing
+// DISABLED-NOT: module 'Swift' was not compiled for testing
+@testable import Swift
 @testable import empty // no-error
 @testable import Testable_ClangModule // no-error
```

Comment:
Now these aren't being checked anymore.

Reply:
How about now? I'm trying to be a bit more explicit than before.